src/components/AttachmentView.js

```diff
@@ -45,10 +49,13 @@ const AttachmentView = (props) => {
     if (Str.isPDF(props.sourceURL)
         || (props.file && Str.isPDF(props.file.name || props.translate('attachmentView.unknownFilename')))) {
         return (
+
```

Comment:
I don't think these empty spaces should be here, it feels inconsistent with our style.